app/assets/stylesheets/error-page.scss

```diff
@@ -1,9 +1,3 @@
-$govuk-compatibility-govuktemplate: true;
-$govuk-use-legacy-palette: false;
-
-@import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/component_support";
-@import "govuk_publishing_components/components/cookie-banner";
-@import "govuk_publishing_components/components/button";
-@import "govuk_publishing_components/components/feedback";
-@import "govuk_publishing_components/components/action-link";
+
```

Comment:
I think we might still need the cookie banner support - if the first page you land on on gov.uk fails for some reason you currently still get the banner served, so we probably need to keep it the same.

Reply:
Good spot, thank you!

Reply:
I had a check and the cookie banner works, phew. 

The cookie banner styles come from [`application.scss`](https://github.com/alphagov/static/blob/5f423d27fb90e90010604404dbd113034f47e1a6/app/assets/javascripts/application.js#L10):
https://github.com/alphagov/static/blob/5f423d27fb90e90010604404dbd113034f47e1a6/app/assets/javascripts/application.js#L10

The `error-page.scss` should only contain components and styles that are needed on the error pages but aren't in the application stylesheet.